examples/2-handle-prompt/index.js

```diff
@@ -0,0 +1,111 @@
+import { createServer } from 'node:http';
+import { Octokit } from "@octokit";
+import { 
+    createAckEvent, 
+    createDoneEvent, 
+    createTextEvent, 
+    parseRequestBody, 
+    prompt,
+    verifyRequestByKeyId
+ } from "@copilot-extensions/preview-sdk";
+
+
+// Define the handler function
+const handler = async (request, response) => {
+
+    if (request.method !== 'POST') {
+        // Handle other request methods if necessary
+        response.writeHead(405, { 'Content-Type': 'text/plain' });
+        console.log(`Method ${request.method} not allowed`);
+        
+        response.end('Method Not Allowed');
+    }
```

Comment:
we need to return here, otherwise the code below will get executed. That's what I meant with return early
```suggestion
        response.end('Method Not Allowed');
        return;
    }
```